packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5147,6 +5148,14 @@ export default class Meeting extends StatelessWebexPlugin {
       this.wirelessShare = true;
     }
 
+    const link = options.startMeetingLink || options.joinMeetingLink;
```

Comment:
Here, we give priority to `startMeetingLink`. I understand from the ticket that the user can give either `startMeetingLink` (only in case of host, I believe) or `joinMeetingLink` (non host and host).
Hence, I decided to give priority to start here - otherwise since both of these are optional, users have flexibility to supply any one of them.